includes/html/pages/pseudowires.inc.php

```diff
@@ -39,7 +39,8 @@
 foreach (dbFetchRows('SELECT * FROM pseudowires AS P, ports AS I, devices AS D WHERE P.port_id = I.port_id AND I.device_id = D.device_id ORDER BY D.hostname,I.ifDescr') as $pw_a) {
     $pw_a = cleanPort($pw_a);
     $i = 0;
-    while ($i < count($linkdone)) {
+//    while ($i < count($linkdone)) { //php8 needed this line changed -st
```

Comment:
Please remove the commented code.  Your commit message will explain your changes.